tests/lib/tools/journal-state

```diff
@@ -40,22 +40,50 @@ check_log_started(){
 }
 
 match_log(){
-    expression=$1
-    shift
-    for _ in $(seq 10); do
+    local wait=1
+    local attempts=10
```

Comment:
IMHO the defaults should be wait=0 and attempts=1 (meaning we just match current log state without sleep/retry).

Reply:
Nice, thanks for the review